web/pdf_thumbnail_viewer.js

```diff
@@ -87,7 +87,7 @@ var PDFThumbnailViewer = (function PDFThumbnailViewerClosure() {
       if (selected) {
         selected.classList.remove('selected');
       }
-      var thumbnail = document.getElementById('thumbnailContainer' + page);
+      var thumbnail = document.querySelector('div.thumbnail[data-page-number="' + page + '"]');
```

Comment:
This line exceeds the maximum length of 80 characters, which causes a linting failure. To resolve this, please change it to:

```javascript
var thumbnail = document.querySelector(
  'div.thumbnail[data-page-number="' + page + '"]');
```